src/styles/nav.scss

```diff
@@ -0,0 +1,27 @@
+.nav {
+  margin-right: 50px;
```

Comment:
BEM block shouldn't have any margins, because in this case, you can't reuse it in other places.
You could specify spacing with paddings to the parent element